var/spack/repos/builtin/packages/glib/package.py

```diff
@@ -134,6 +136,7 @@ class Glib(MesonPackage, AutotoolsPackage):
     )
 
     with when("build_system=meson"):
+        depends_on("meson@1.2.0:", when="@2.79:", type="build")
         depends_on("meson@0.60.0:", when="@2.73:", type="build")
```

Comment:
This should probably become `when="@2.73:2.78"` then.

Reply:
```suggestion
        depends_on("meson@0.60.0:", when="@2.73:2.78", type="build")
```